nautobot_ssot/migrations/0007_replace_dashed_custom_fields.py

```diff
@@ -0,0 +1,58 @@
+from django.db import migrations
+from nautobot.dcim.models import Device, DeviceType, Interface, Manufacturer, Location
+from nautobot.extras.models import Role
+from nautobot.extras.utils import FeatureQuery
+from nautobot.ipam.models import IPAddress, VLAN
+
+CF_KEY_CHANGE_MAP = {
+    "ssot_synced_to_servicenow": "ssot-synced-to-servicenow",
+    "last_synced_from_sor": "ssot-synced-from-ipfabric",
+    "ipfabric_site_id": "ipfabric-site-id",
+    "ssot_synced_to_infoblox": "ssot-synced-to-infoblox",
+}
+
+
+def replace_dashed_custom_fields(apps, schema_editor):
+    CustomField = apps.get_model("extras", "customfield")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+
+    for new_key, old_key in CF_KEY_CHANGE_MAP.items():
+        if not CustomField.objects.filter(key=new_key).exists():
+            for custom_field in CustomField.objects.filter(key=old_key):
+                original_key = custom_field.key
+                print(
+                    f'   CustomField instance "{custom_field.label}" key attribute "{original_key}" is being changed to "{new_key}".'
+                )
+                custom_field.key = new_key
+                custom_field.save()
+
+    for ct in ContentType.objects.filter(FeatureQuery("custom_fields").get_query()):
```

Comment:
Doesn't look like `ct` is actually used anywhere - unnecessary outer `for` loop?

Reply:
Good catch. You're right, we're not using that anymore so removed.